### PR TITLE
add SimpleFSSyncStatus

### PR DESCRIPTION
**Pull request by songgao**
Needed by and uses updated vendor from https://github.com/keybase/client/pull/11186.

### CODE REVIEW

File: simplefs/simplefs.go [hunk 1]
```diff
@@ -1296,3 +1296,18 @@ func (k *SimpleFS) SimpleFSDumpDebuggingInfo(ctx context.Context) error {
 	k.idd.ForceDump(ctx)
 	return nil
 }
+
+// SimpleFSSyncStatus - Get sync status.
+func (k *SimpleFS) SimpleFSSyncStatus(ctx context.Context) (keybase1.FSSyncStatus, error) {
+	jServer, err := libkbfs.GetJournalServer(k.config)
```
Comment (strib) [inline simplefs/simplefs.go]: Can you please do this at the top, so we can get a debug ID?

```go
    ctx = k.makeContext(ctx)
```

File: vendor/vendor.json [hunk 2]
```diff
@@ -238,10 +238,10 @@
 			"revisionTime": "2018-01-25T22:56:34Z"
 		},
 		{
-			"checksumSHA1": "Jm51R9I7AKovceb8ZWXaBkLQCOE=",
+			"checksumSHA1": "grS9oX+0LtDbt/znfITZ5/CPc98=",
 			"path": "github.com/keybase/client/go/protocol/keybase1",
-			"revision": "97192f698f4d04d79a4b96fa73ac526aa6c20af7",
-			"revisionTime": "2018-03-20T23:57:06Z"
+			"revision": "2119e7ce2550b1e542dc6498d6710ef1a4b36838",
```
Comment (strib) [inline vendor/vendor.json]: I guess you'll fix this commit hash up after commiting keybase/client#11186?  Fine with me if you want to commit this first, then the other one, then fix this up in a new commit.

Comment (songgao) [reply vendor/vendor.json]: I'll merge this one now, hoping to make client CI pass, merge CI, then make another commit here to update the hash.

Comment (songgao) [reply vendor/vendor.json]: OK done!

**Pull request merged**